addons/mail/static/src/components/composer/composer.xml

```diff
@@ -140,7 +140,7 @@
     </t>
 
     <t t-name="mail.Composer.actionButtons" owl="1">
-        <div class="o_Composer_actionButtons" t-att-class="{ 'o-composer-is-compact': composerView.isCompact }">
```

Comment:
There seems to be some unrelated changes across the diff so make sure to check everything carefully again.